normandy/settings.py

```diff
@@ -151,6 +151,7 @@ class Development(Base):
 
 class Production(Base):
     """Settings for the production environment."""
+    SECURE_PROXY_SSL_HEADER = values.Value(None)
```

Comment:
This needs to be a `TupleValue`. [docs](http://docs.aws.amazon.com/ElasticLoadBalancing/latest/DeveloperGuide/x-forwarded-headers.html#x-forwarded-proto).